app/src/scripts/dashboard/dashboard.jobs.store.js

```diff
@@ -56,8 +56,9 @@ let DashboardJobStore = Reflux.createStore({
                 isTimestamp: true
             },
             sortOptions: [
-                {label: 'Name', property: 'datasetLabel'},
-                {label: 'Date', property: 'analysis.created', isTimestamp: true}
+                {label: 'Name', property: 'jobName'},
+                {label: 'Date', property: 'analysis.created', isTimestamp: true},
+                {label: 'Dataset', property: 'datasetLabel'},
```

Comment:
The linter doesn't like this because of the extra comma at the end of the list. You can see the linting issues before you commit with 'yarn lint'.

> /srv/crn-app/src/scripts/dashboard/dashboard.jobs.store.js
  61:61  error  Unexpected trailing comma  comma-dangle
> ✖ 1 problem (1 error, 0 warnings)

Reply:
I have removed the rogue comma